keras_cv/layers/preprocessing3d/global_random_dropping_points.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing3d import base_augmentation_layer_3d
+
+POINT_CLOUDS = base_augmentation_layer_3d.POINT_CLOUDS
+BOUNDING_BOXES = base_augmentation_layer_3d.BOUNDING_BOXES
+
+
+class GlobalRandomDroppingPoints(base_augmentation_layer_3d.BaseAugmentationLayer3D):
+    """A preprocessing layer which randomly drops point during training.
+
+    This layer will randomly drop points based on keep_probability.
+    During inference time, the output will be identical to input. Call the layer with `training=True` to drop the input points.
+
+    Input shape:
+      point_clouds: 3D (multi frames) float32 Tensor with shape
+        [num of frames, num of points, num of point features].
+        The first 5 features are [x, y, z, class, range].
+      bounding_boxes: 3D (multi frames) float32 Tensor with shape
+        [num of frames, num of boxes, num of box features].
+        The first 7 features are [x, y, z, dx, dy, dz, phi].
+
+    Output shape:
+      A dictionary of Tensors with the same shape as input Tensors.
+
+    Arguments:
+      drop_rate: A float scalar or Tensor sets the probability threshold for dropping the points.
+    """
+
+    def __init__(self, drop_rate, **kwargs):
+        super().__init__(**kwargs)
+        keep_probability = 1 - drop_rate
+        if keep_probability < 0:
+            raise ValueError("keep_probability must be >=0.")
```

Comment:
nit: since `drop_rate` is what's in the API, maybe this should say "drop rate must be <= 1"

Reply:
Good point. Done.